app/javascript/components/search/controls/KeywordSearch.jsx

```diff
@@ -35,6 +35,7 @@ export default function KeywordSearch({ keywordPrompt }) {
       <p>&quot;single cell&quot; Smith</p>
     </ul>
     <p>* Structured data for authors is new in SCP, and many studies lack it, so author search results may be limited.</p>
+    <p>You can find studies directly by entering their accessions as terms: SCP1 SCP101 SCP1234</p>
```

Comment:
Thank you for adding this! Should we include "study accession" in the first sentence of this infobox as well?

Reply:
Actually that's a great idea - I'll do that.